### ISSUE TITLE
Channel Name via Serial.Print

### ISSUE THREAD

**Issue by chefseg** [hyOzd/serialplot#65]
I downloaded version SerialPlot v0.12 (Windows executable install file) from [https://hackaday.io/project/5334-serialplot-realtime-plotting-software/log/192838-serialplot-v012-release]. What is the recommended format to print in the Arduino sketch to dynamically change the channel name? I tried using Serial.print("Channel1:"); Serial.println(value); but no success. Any help would be appreciated!

**Comment by hyOzd** [comment]
Can you post an example of your output from a text console?

**Comment by chefseg** [comment]
Here is an example of one line of the serial output "Roll: 30.1°, Pitch: 10.1°, Yaw: 0.0°"

**Comment by hyOzd** [comment]
You can't have degree symbol in your output. Parser doesn't recognize that.

**Comment by chefseg** [comment]
Even without the degree symbols it still doesn't work. Recall, I am using the desktop version that is downloadable via link:  [https://hackaday.io/project/5334-serialplot-realtime-plotting-software/log/192838-serialplot-v012-release]

Are you certain that version contains the code to auto display the tags to the channels? From my recollection, it appears that there was an update on Git to support this feature but this version was released at an earlier date, reference this: **[Allow Arduino Serial Plotter style ascii data #43](https://github.com/hyOzd/serialplot/pull/43)**

**Comment by hyOzd** [comment]
Yeah sorry, you are right. v0.12 is quite old. It doesn't include relevant commit. You will have to build from source to have that feature, sorry.

**Comment by CeivenLean** [comment]
> Yeah sorry, you are right. v0.12 is quite old. It doesn't include relevant commit. You will have to build from source to have that feature, sorry.

is there any possible that a new version will be released

**Comment by s-light** [comment]
should be closed with current available release.
